EfficientNet example notebook

AshishSardana opened the pull request:
# Description

Added an example notebook for EfficientNet-B0 image classification model.

## Type of change

Please delete options that are not relevant and/or add your own.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [x] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

narendasan reviewed (commented):
Can you list the notebook in https://github.com/NVIDIA/Torch-TensorRT/blob/master/docsrc/index.rst#getting-started so it is included in the documentation?

narendasan commented:
Also please sign your commits

AshishSardana commented:
I've listed the notebook and signed my latest commit. Would that be good to go?

narendasan commented:
You need to sign all commits. See https://github.com/NVIDIA/Torch-TensorRT/pull/709/checks?check_run_id=4214564245 for instructions

narendasan commented on docsrc/index.rst:
```diff
@@ -53,6 +53,7 @@ Getting Started
    _notebooks/Resnet50-example
    _notebooks/ssd-object-detection-demo
    _notebooks/vgg-qat
+   _notebooks/EfficientNet-example.ipynb 
```
you dont need the .ipynb

AshishSardana commented:
Signed all commits and fixed the ".ipynb" typo.

narendasan commented:
Why are a bunch of files getting recommitted?

AshishSardana commented:
When I signed off my previous commits using DCO's suggested command - `git rebase HEAD~5 --signoff` - I think it signed some of your commits as well:
```First, rewinding head to replay your work on top of it...
Applying: add EfficientNet example notebook
Applying: updated documentation
Applying: updated preproc
Applying: chore: Bump version
Applying: fix: Update base container for dockerfile
Applying: fix(//py): Constrain the CUDA version in container builds
Applying: refactor: Update the CI deps
Applying: chore: update the container
Applying: Update network.py
Applying: Update elu_model.py
Applying: test: Add a test for the global compile function
Applying: test(//py): Test nn.module compilation
Applying: refactor: CI updates
Applying: refactor: Try setting permisions for job
Applying: refactor: Apply linting
Applying: Update linter.yml
Applying: fix: Missing log severity
Applying: Update README.md
Applying: fix(//cpp/bin/torchtrtc): Fix mbs
Applying: chore(//docsrc): Update docs deps to fix docgen
Applying: updated index.rst```

Would you suggest that I raise a fresh PR instead?

narendasan commented:
yeah probably best to do that

AshishSardana commented:
I've created a new PR here - https://github.com/NVIDIA/Torch-TensorRT/pull/723 - and hence closing this.
